form-editor-cae/src/main/java/com/tallence/formeditor/cae/FormFreemarkerFacade.java

```diff
@@ -32,7 +33,11 @@
 
   private final FormElementFactory formElementFactory;
 
-  public FormFreemarkerFacade(FormElementFactory formElementFactory) {
+  @Inject
```

Comment:
Die Annotation ist hier falsch. Die Klasse wird aus der XML-Config instanziiert. Stattdessen sollte der Service in https://github.com/tallence/core-forms/blob/master/form-editor-cae/src/main/resources/META-INF/coremedia/form-freemarker-views.xml nachgetragen werden.